recipes/mpiFFT4py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mpiFFT4py" %}
```

Comment:
Not for me to answer, but it's not a big deal for this package. And others seem to manage just fine (e.g., PETSc, SLEPc, BLAS).